src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -136,7 +151,44 @@ def get(self, svc_id):
         daemon['rgw_metadata'] = metadata
         daemon['rgw_status'] = status
         return daemon
-
+    
+    @RESTController.Collection(method='GET', path='/multisite')
+    @allow_empty_body
+    def get_multisite_info(self):
+        multisite_info = []
+        try:
+            instance = RgwClient.admin_instance()
+        except NoRgwDaemonsException:
+            return multisite_info
```

Comment:
Exceptions should not just returned. It should be raised properly to the dashboard. Otherwise this will just crash in the backend with some tracebacks.

Instead it should be raised to dashboard
```
```suggestion
            return multisite_info
```
```suggestion
        except NoRgwDaemonsException as error:
           raise DashboardException(
                           error, 
                           http_status_code=404, 
                           component='rgw')
```